src/utils/formatters.js

```diff
@@ -4,9 +4,24 @@ import snxJSConnector from './snxJSConnector';
 export const formatCurrency = (value, decimals = 2) => {
 	if (!value) return 0;
 	if (!Number(value)) return 0;
+	if (Number.isInteger(value)) return numbro(value).format('0,0');
 	return numbro(value).format('0,0.' + '0'.repeat(decimals));
 };
 
+export const formatCurrencyWithPrecision = value => {
+	return formatCurrency(value, getPrecision(value));
+};
+
+const getPrecision = amount => {
+	if (amount >= 1) {
+		return 2;
+	} else if (amount > 0.01) {
+		return 4;
+	} else return 6;
+};
+
+window.formatCurrencyWithPrecision = formatCurrencyWithPrecision;
+window.getPrecision = getPrecision;
```

Comment:
whoa - why are you attaching to `window`???

Reply:
My bad. That was to test something in the JS console.